### DISCUSSION TITLE
Welcome to nano Discussions!

### DISCUSSION THREAD

**Discussion by lonng** [lonng/nano#77] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by blkscales** [comment]
Hello, may ask some questions on installation of nano?

For installation, can I use 
go install github.com/lonng/nano@latest   
to replace
go get github.com/lonng/nano
?

It seems we cannot use 'go get' command after go version 1.8  .
And how can we test the nano is installed properly ?
